src/platforms.c

```diff
@@ -85,48 +126,19 @@ platforms_t *load_platforms_from_file(const char *filename)
 {
```

Comment:
Rename `load_platforms_from_file` -> `create_platforms_from_file` and put it along with other platforms constructors